docs/kubernetes/AWSIntegration.md

```diff
@@ -0,0 +1,145 @@
+# Integration with an AWS Kubernetes cluster
+This guide will walk you through how to use Calico Networking with an existing AWS-Kubernetes cluster.
+
+## Requirements
+* A working Kubernetes Deployment on AWS with one master and at least one minion
+    - While any AWS deployment will do, we recommend the Kubernetes [`kube-up` guide](https://github.com/kubernetes/kubernetes/blob/release-1.0/docs/getting-started-guides/aws.md) for AWS. This guide was created with the `kube-up` script in mind.
+* SSH access to your master and minions
+    - Unless otherwise specified, the `kube-up` script will create a `kube_aws_rsa` private key in the `~/.ssh` folder which you can use to access your AWS Instances.
+    - SSH in with the following command `ssh -i </path/to/key> ubuntu@<PUBLIC_IP>`
+
+## Preparing your master services
+On your master, download our etcd manifest
+```
+wget https://raw.githubusercontent.com/projectcalico/calico-kubernetes-coreos-demo/ah3-config-update/manifests/calico-etcd.manifest
+```
+Replace all instances of `<PRIVATE_IPV4>` with your master's IP or hostname. Then, place the manifest file in the `/etc/kubernetes/manifests/` directory. 
+```
+sudo mv calico-etcd.manifest /etc/kubernetes/manifests/
+```
+After a short delay, the kubelet on your master will automatically create a container for the new etcd which can be accessed on port 6666 of your master.
+
+## Running `calico-node`
+On each of your AWS Instances, perform the following steps.
+
+### Install calicoctl and calico_kubernetes
+* Download and install the `calicoctl` binary
+```
+wget https://github.com/projectcalico/calico-docker/releases/download/v0.7.0/calicoctl
+chmod +x calicoctl
+sudo mv calicoctl /usr/bin/
+```
+
+* Download and install the plugin binary
+```
+wget https://github.com/projectcalico/calico-kubernetes/releases/download/v0.2.1/calico_kubernetes
+chmod +x calico_kubernetes
+sudo mkdir -p /usr/libexec/kubernetes/kubelet-plugins/net/exec/calico/
+sudo mv calico_kubernetes /usr/libexec/kubernetes/kubelet-plugins/net/exec/calico/calico
+```
+
+* Run Calico Node
+```
+sudo modprobe ipip
+sudo ETCD_AUTHORITY=<MASTER_PRIVATE_IPV4>:6666 calicoctl node
+```
+Set up an IP pool with IP-in-IP enabled. This is a  [necessary step](https://github.com/projectcalico/calico-docker/blob/20adfd2b7640af9d85c4af76916e043286691452/docs/FAQ.md#can-i-run-calico-in-a-public-cloud-environment) in any public cloud environment.
```

Comment:
You can fix up that URL to use a released version of the doc, rather than a commit hash:

https://github.com/projectcalico/calico-docker/blob/v0.7.0/docs/FAQ.md#can-i-run-calico-in-a-public-cloud-environment